pkg/controllers/uiplugin/components.go

```diff
@@ -166,6 +216,33 @@ func newDeployment(info UIPluginInfo, namespace string) *appsv1.Deployment {
 	return plugin
 }
 
+func computeConfigMapHash(cm *corev1.ConfigMap) string {
+	h := sha256.New()
```

Comment:
I'd rather use hash/fnv to make it clear that we need a non-crypto hashing function.